gateway/modules/filestore/store.go

```diff
@@ -46,13 +48,13 @@ type FileStore interface {
 	CreateFile(req *model.CreateFileRequest, file io.Reader) error
```

Comment:
Pass context in each function. Use that context in gcp. Make sure you increate the context timeout to 30 mins in the api handlers of file storage apis

Reply:
30 mins are your sure, or is this is a type 30 secs you mean